packages/codemirror/src/tokens.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { Token } from '@lumino/coreutils';
+
+import CodeMirror from 'codemirror';
+
+/* tslint:disable */
+/**
+ * The CodeMirror token.
+ */
+export const ICodeMirror = new Token<ICodeMirror>(
+  '@jupyterlab/codemirror:ICodeMirror'
+);
+/* tslint:enable */
+
+/** The CodeMirror interface. */
+export interface ICodeMirror {
+  /**
+   * A singleton CodeMirror module, rexported.
+   */
+  codemirrorSingleton(): typeof CodeMirror;
```

Comment:
This is now a property?

Reply:
Yeah, trying to get a local build setup, having issues with `canvas` compiling so I can lint/test to avoid more icky commits...

Reply:
renamed to `CodeMirror`, made a property...